packages/ui-components/src/globalStyle.tsx

```diff
@@ -10,27 +10,28 @@ const GlobalStyle = createGlobalStyle`
   }
 
   body {
-    height: 100%;
-    width: 100%;
+    minHeight: 100%;
+    minWidth: 100%;
     display: flex;
     justify-content: center;
+    align-items: center;
     margin: 0;
     padding: 0;
     font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans',
       'Droid Sans', 'Helvetica Neue', sans-serif;
     -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
   }
 `;
 
+// ordered darkest to lightest
 const substrateLightTheme = {
   black: '#222',
-  cardBorder: '#f2f2f2',
-  darkBlue: '#5c53fc',
   grey: '#888',
-  lightBlue1: '#53a0fd',
-  lightBlue2: '#51a0ec',
   purple: '#8479f3',
+  darkBlue: '#5c53fc',
+  lightBlue2: '#51a0ec',
+  lightBlue1: '#53a0fd',
```

Comment:
Could this be renamed into something more generic like `primary`, `primaryDark`, `primaryLight` and `secondary` instead of `darkBlue `, `blue` etc? It's very likely to be shades of `pink` in the polkadot theme (100% not blue), so any name that includes blue would be highly confusing.

Related to #136

Reply:
Regarding color schemes, it's a bit tricky because this UI is meant for the chains running on substrate and particulary for light client users on that, which could conceivably be a polkadot parachain but not necessarily. Whereas apps is meant for all Polkadot chains and currently 
that's just the Relay chain which is actually built on Substrate itself...

So considering this UI at least initially will cater for standalone Substrate chains, having the Polkadot theme would be confusing and inaccurate. But considering Substrate chains are meant to be easily onramped into Polkadot, I think it makes sense to have that theme available that 'activates' if the current chain is a parachain.

it's confusing but i think at least for now the focus is on standalone substrate chains.

![Screen Shot 2019-03-14 at 3 55 28 PM](https://user-images.githubusercontent.com/10432070/54340200-daf13380-4671-11e9-9386-eb6b574d5241.png)

Reply:
Ah, makes total sense! Feel free to close the #136 ticket, then.

But in general it would be nice to just have 2 color schemes as objects, one for polkadot and one for substrate, which we can just import into our projects and which are easily replaceable, if people want to use a custom theme. That way we'd only have to maintain the colors in one place.

I know that Maciej wants to untie Telemetry visually from Polkadot. He wants to turn it into more generic React component to make it re-usable/embeddable, so he's going to need themes. Same for Polkadot Apps, we're also using themes there. 

I'll write a ticket in https://github.com/polkadot-js/dev for, that's probably the best place for it, @jacogr ?

Reply:
Regardless of themes, is there a better name for these 2 colors other than 1 and 2? extraLightBlue? skyBlue?